pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2645,6 +2740,39 @@
       %-  =<  request-is-logged-in:authentication
           (per-server-event [eny *duct now rof] server-state.ax)
       %*(. *request:http header-list ['cookie' u.cookies]~)
+    ::
+        [%cache @ @ ~]
+      ?~  aeon=(slaw %u i.t.tyl)         [~ ~]
+      ?~  url=(slaw %t i.t.t.tyl)        [~ ~]
+      ?~  entry=(~(get by cache) u.url)  [~ ~]
+      ?.  =(u.aeon aeon.u.entry)         [~ ~]
+      ?~  val.u.entry                    [~ ~]
+      =/  val  u.val.u.entry
+      ?:  ?=(%payload -.body.val)
+        ``noun+!>(val)
+      ::  body is a scry; scry it to get a payload
+      ::
+      ?~  path.body.val                  [~ ~]
+      ?~  bem=(de-beam t.path.body.val)  [~ ~]
+      =/  res=(unit (unit cage))  (rof ~ i.path.body.val u.bem)
+      ?~  res    [~ ~]
+      ?~  u.res  [~ ~]
+      =*  vase   q.u.u.res
+      ?.  ?=(@ q.vase)  [~ ~]
+      =/  data  (as-octs:mimes:html q.vase)
+      =/  =simple-payload:http
+        :-  :-  status-code=200
+          ^=  headers
+            :~  ['content-type' 'text/html']
+                ['content-length' (crip (format-ud-as-integer p.data))]
+            ==
+        data=[~ data]
+      =/  cache-entry
+        :*
+          auth=auth.val
+          body=[%payload simple-payload]
+        ==
+      ``noun+!>(val)
```

Comment:
the scry still returns a `cache-entry`, but the body will always be a `%payload`. That feels a little off to me, but it didn't seem worth it to define a separate `cache-scry-result` type. Thoughts?